src/routers/alpha-router/alpha-router.ts

```diff
@@ -1213,6 +1213,7 @@ export class AlphaRouter
       trade,
       methodParameters,
       blockNumber: BigNumber.from(await blockNumber),
+      hitsCachedRoute: swapRouteFromChain === null,
```

Comment:
`!swapRouteFromChain && swapRouteFromCache` ?

Reply:
Sure. They seems mutually exclusive, but can be safer this way.

Reply:
It has to be `!swapRouteFromChain && swapRouteFromCache !== null` to be type conformant.